arcgis-ios-sdk-samples/Augmented reality/Display scenes in tabletop AR/README.md

```diff
@@ -28,6 +28,7 @@ You'll see a feed from the camera when you open the sample. Tap on any flat, hor
 
 * ArcGISARView
 * AGSSceneView
+* AGSSurface
```

Comment:
Same question for adding API to README and the problem with metadata.

Reply:
We'll tackle this separately as part of issue 1705. So don't worry about it for now.